README.md

```diff
@@ -0,0 +1,118 @@
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+Hello Microverse projects shows the concepts I have understood
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+The project is to determine the basic knowledge one has in git,HTML5,CSS and Linters
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML5</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- None
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] You can rremove this live demo section completely since you are not required to deploy the project. Only add the live demo when you have the project deployed 😄